src/plugins/image_display/ImageDisplay.cc

```diff
@@ -184,6 +184,22 @@ void ImageDisplay::ProcessImage()
       common::Image::ConvertToRGBImage<uint8_t>(
           this->dataPtr->imageMsg.data().c_str(), width, height, output);
       break;
+    case msgs::PixelFormatType::BAYER_RGGB8:
+      common::Image::ConvertToRGBImage<uint8_t>(
+          this->dataPtr->imageMsg.data().c_str(), width, height, output);
+      break;
+    case msgs::PixelFormatType::BAYER_BGGR8:
+      common::Image::ConvertToRGBImage<uint8_t>(
+          this->dataPtr->imageMsg.data().c_str(), width, height, output);
+      break;
+    case msgs::PixelFormatType::BAYER_GBRG8:
+      common::Image::ConvertToRGBImage<uint8_t>(
+          this->dataPtr->imageMsg.data().c_str(), width, height, output);
+      break;
+    case msgs::PixelFormatType::BAYER_GRBG8:
+      common::Image::ConvertToRGBImage<uint8_t>(
+          this->dataPtr->imageMsg.data().c_str(), width, height, output);
+      break;
```

Comment:
since the conversions are all the same, you should be able do a fall-through:

```
    case msgs::PixelFormatType::BAYER_RGGB8:
    case msgs::PixelFormatType::BAYER_BGGR8:
    case msgs::PixelFormatType::BAYER_GBRG8:
    case msgs::PixelFormatType::BAYER_GRBG8:
      common::Image::ConvertToRGBImage<uint8_t>(
          this->dataPtr->imageMsg.data().c_str(), width, height, output);
      break;

```